src/test/java/seedu/address/logic/parser/TaTrackerParserTest.java

```diff
@@ -56,10 +55,10 @@ public void parseCommand_delete() throws Exception {
     @Test
     public void parseCommand_edit() throws Exception {
         Student student = new PersonBuilder().build();
-        EditPersonDescriptor descriptor = new EditPersonDescriptorBuilder(student).build();
-        EditCommand command = (EditCommand) parser.parseCommand(EditCommand.COMMAND_WORD + " "
+        EditStudentCommand.EditStudentDescriptor descriptor = new EditStudentDescriptorBuilder(student).build();
```

Comment:
should just use `EditStudentDescriptor `directly instead of referring to `EditStudentCommand`